src/js/components/molecules/social-sharing/social-sharing.ts

```diff
@@ -0,0 +1,44 @@
+/** @format */
+
+import { Vue, Component, Prop } from 'vue-property-decorator';
+// import { SocialSharingProps } from './social-sharing.models';
+import WithRender from './social-sharing.vue.html';
+import { generateSocialSharingUrl } from '../../../utils/generate-social-sharing-url';
+import BrowserCapabilities from '../../../utils/browser-capabilities';
+
+@WithRender
+@Component({
+    name: 'SocialSharing',
+})
+
+export class SocialSharing extends Vue {
+    // @Prop({ type: Array, required: true }) items!: SocialSharingProps['items'];
```

Comment:
Use the props

Reply:
There are no props… it's hardcoded… for now.

Reply:
Ok cool,l it not managed in the CMS, then please remove the line given it's not needed.